src/serialization/sb2_specmap.js

```diff
@@ -1201,7 +1201,8 @@ const specMap = {
         argMap: [
             {
                 type: 'field',
-                fieldName: 'VARIABLE'
+                fieldName: 'VARIABLE',
+                variableType: ''
```

Comment:
@kchadha do we want to use the `Variable.SCALAR_TYPE` constant here instead of the raw string?

Reply:
I was actually going to pose this question in my actual PR description. I can make the change so that variableType always refers to the static constants we defined.